README.md

```diff
@@ -79,3 +79,12 @@ from the host. So far, so good. The tricky part is being able to run apps on
 the host while they appear from within the docker. The hack used here is to
 use SSH port tunnelling between the container and the host. That's what the JS
 tool `index` is used for.
+
+
+## Troubleshooting
+
+**`sync.sh` hangs when trying to download packages.**
```

Comment:
Sorry, it's `build.sh`. I should'nt try do do more than one thing at the same time. I should drink more coffee.